packages/modules/web_themes/koala/source/src/stores/mqtt-store.ts

```diff
@@ -501,6 +502,7 @@ export const useMqttStore = defineStore('mqtt', () => {
       scale: boolean = true,
       inverted: boolean = false,
       defaultString: string = '---',
+      showDecimalPlaces: boolean = false,
```

Comment:
Um es noch flexibler zu machen, könnte die Anzahl Nachkommastellen übergeben werden.
```suggestion
      decimalPlaces: int = 0,
```

Reply:
Eingabe-Parameter fur Anzahl die Nachkommastellen hinzugefügt.